### PR TITLE
[elastic-query-exporter] Add alert for unexpected VM power down

### PR DESCRIPTION
**Pull request by max-len**
Loglevel INFO, playbook needs to be added, no VM info in alert

### CODE REVIEW

File: prometheus-exporters/elastic-query-exporter/alerts/logs.alerts [hunk 1]
```diff
@@ -41,3 +41,16 @@ groups:
         annotations:
           description: "VCSA logs missing in Octobus for vCenter {{ $labels.vcenter }}."
           summary: "VCSA logs missing in Octobus for vCenter {{ $labels.vcenter }}."
+      - alert: VMPoweredDownDuringFailedHostMigration
+        expr: elasticsearch_octobus_vmdown_doc_count
+        for: 5m
+        labels:
+          severity: info
+          tier: vmware
+          service: audit
```
Comment (christopherhans) [inline prometheus-exporters/elastic-query-exporter/alerts/logs.alerts:50]: Ne service is not `audit`, it is `compute`.
We used audit for the audit related alerts.

File: prometheus-exporters/elastic-query-exporter/files/exporter.cfg [hunk 2]
```diff
@@ -156,3 +155,24 @@ QueryJson = {
...
+          { "exists": { "field": "node.nodename" }},
+          { "match_phrase": { "message": "An operation required the virtual machine to quiesce and the virtual machine was unable to continue running." }}
+        ],
+        "filter": [
+          { "range": { "@timestamp": { "gte": "now-1h" }}}
+        ]
+      }
+    },
+    "aggs": {
+      "hostsystem": { "terms": { "field": "node.nodename.keyword" }}
```
Comment (christopherhans) [inline prometheus-exporters/elastic-query-exporter/files/exporter.cfg:176]: can't we just use `message` as field?

**Review by christopherhans — CHANGES_REQUESTED** [review]

**Comment by christopherhans** [comment]
If we could get the whole message in the label `message`, we could do something like
`{{ labels.message | regex_replace('.+', '(?<=on\s).+\s\(.+\)' }}`
this will result in something like `cdd23v010062 (c52b98d6-af94-404a-afb8-619734b062e4)`

**Comment by christopherhans** [comment]
Playbook: https://operations.global.cloud.sap/docs/devops/alert/vcenter/#VM%20powered%20off%20during%20vMotion

File: prometheus-exporters/elastic-query-exporter/alerts/logs.alerts (hunk 1 above)
Comment (max-len) [reply prometheus-exporters/elastic-query-exporter/alerts/logs.alerts]: Fixed

**Comment by max-len** [comment]
> Playbook: https://operations.global.cloud.sap/docs/devops/alert/vcenter/#VM%20powered%20off%20during%20vMotion

Added

File: prometheus-exporters/elastic-query-exporter/files/exporter.cfg (hunk 2 above)
Comment (max-len) [reply prometheus-exporters/elastic-query-exporter/files/exporter.cfg:176]: Unfortunately the mapped keyword field doesn't behave consistently on ElasticDB side. In the sample alert I was testing with this field was "missing" and the aggregate returned no results.

Comment (christopherhans) [reply prometheus-exporters/elastic-query-exporter/files/exporter.cfg:176]: I just don't think that only pointing out the host is a huge help, if there are dozens of powered off VMs.

Could you please reach out to OCB Team, why this field behaves differently?

Comment (max-len) [reply prometheus-exporters/elastic-query-exporter/files/exporter.cfg:176]: Yes, I'm on it. I'd suggest to merge this to test the current state in QA in parallel.

**Review by christopherhans — APPROVED** [review]
Roll out for testing in info severity.

**Pull request merged**